spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2660,4 +2660,19 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("array_intersect") {
```

Comment:
The PR title says `array_remove` but the test is for `array-intersect`?

Or other places that we already have tests?